ui/mirage/scenarios/default.js

```diff
@@ -575,6 +575,14 @@ function smallCluster(server) {
     volume.save();
   });
 
+  server.create('dynamic-host-volume', {
+    name: 'dynamic-host-volume',
+    namespaceId: 'default',
+    createTime: new Date().getTime() * 1000000,
+    modifyTime: new Date().getTime() * 1000000,
+    allocations: csiAllocations,
```

Comment:
Fine that we're reusing allocs for CSI here of course, but I noticed on the preview that the CSI volumes don't actually show the write allocs / read allocs. Ex. https://nomad-dnblzzfqu-hashicorp.vercel.app/ui/storage/volumes/csi/port-4@namespace-1

Reply:
This is a good point, and not sure why those don't show up there. Hoping they're not mutually exclusive in mirage or something.